### DISCUSSION TITLE
"Upload proof of your academic status" page won't initiate an application for me

### DISCUSSION THREAD

**Discussion by iamjonjackson** [community/community#111137] [GitHub Education] [answered]
### Hi everyone,

I can't seem to get the "Upload proof of your academic status" page to submit and accept my evidence.

Every time I upload an image (correct size and dimensions) and submit the form, the page just reloads again with no error or feedback. 

I've tried in 2 browsers (Brave with shields down, and Safari). I am on MacOS, on a MacBook Pro M1 13"

URL is: https://education.github.com/discount_requests/<8-digit-ID>/additional_information

Any suggestions on what to do or what might be causing this?

Thanks!

**Comment by iamjonjackson** [comment]
Anyone?

**Comment by amankr1469** [comment, marked as answer]
Had this same issue. If you look at the browser console you may see an error there. I had an issue trying to submit my transcript and was getting a 413 Payload too Large POST error. You should try to resize your screenshots if you submitted any.

**Reply by iamjonjackson** [reply]
Good shout, thank you. That was the issue. Frustrating their UI says they accept files up to 1MB and the file I was trying to upload was only 750kb.

**Reply by amankr1469** [reply]
Glad it helped haha
